benchmarks/benchmark_result.py

```diff
@@ -14,6 +14,7 @@ class BenchmarkResult:
     family: Optional[str] = attr.ib(default=None)
     cost: Optional[float] = attr.ib(default=None, metadata={"aggregation": "sum"})
     tokens: Optional[int] = attr.ib(default=None, metadata={"aggregation": "average"})
+    count: int = attr.ib(default=1, metadata={"aggregation": "sum"})
```

Comment:
Previously when the summary was with the run we could compute the length of results but now the summary needs another way to be aware of it.